tests/git-ssh-server/setup/Earthfile

```diff
@@ -0,0 +1,90 @@
+VERSION 0.6
```

Comment:
this file was moved from `tests/git-ssh-server/Earthfile` to `tests/git-ssh-server/setup/Earthfile`